src/Controller/ThrowHttpException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of the Mediapart LaPresseLibre Bundle.
+ *
+ * CC BY-NC-SA <https://github.com/mediapart/lapresselibre-bundle>
+ *
+ * For the full license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Mediapart\Bundle\LaPresseLibreBundle\Controller;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+trait ThrowHttpException
```

Comment:
Seemes useless to me, i'll prefer directly throw the exception in the class, we don't need a trait to share 1 line of code IMO